Arduino/Propulsion/src/PropulsionModule.cpp

```diff
@@ -1,6 +1,11 @@
 #include <Arduino.h>
+#ifdef TINYDUINO
```

Comment:
Is there are more explicit way to group these, brackets or similar? I found it hard to follow at first. Perhaps if nothing exists we can indent the conditional lines to make it more apparent?

Reply:
I don't often see preprocessor directives indented, so I'm hesitant to do that. I could add extra newlines?

```cpp
#include <Arduino.h>

#ifdef TINYDUINO

#include <Wire.h>
#include <MotorDriver.h>

#else

#include <SoftwareSerial.h>
#include <PololuQik.h>

#endif

#include <Crc16.h>
```

Reply:
That said, preprocessor directives are generally weird and initially-hard-to-follow (and some may say that C++ as a language is generally so).

Reply:
New Lines are good enough.